src/pages/resultsView/download/DownloadTab.tsx

```diff
@@ -204,6 +206,18 @@ export default class DownloadTab extends React.Component<IDownloadTabProps, {}>
         invoke:()=>Promise.resolve(this.alteredSamples.result!.join("\n"))
     });
 
+    readonly unalteredSamples = remoteData<string[]>({
+        await:()=>[this.caseAlterationData],
+        invoke:()=>Promise.resolve(this.caseAlterationData.result!
+            .filter(caseAlteration => !caseAlteration.altered)
+            .map(caseAlteration => `${caseAlteration.studyId}:${caseAlteration.sampleId}`))
+    });
+
+    readonly unalteredSamplesText = remoteData<string>({
+        await: ()=>[this.unalteredSamples],
+        invoke:()=>Promise.resolve(this.unalteredSamples.result!.join("\n"))
```

Comment:
again, just do this in logic.  i don't think there's a strong reason to encapsulate this simple operation inside a promise